00 - Notes/อินเทอร์เน็ตคืออะไร.md

```diff
@@ -1,3 +1,2 @@
 # Internet คืออะไร ?
-**Internet** คือระบบเครือข่ายคอมพิวเตอร์ขนาดใหญ่ ที่ประกอบไปด้วยเครือข่ายย่อยทั้งจากส่วนบุคคลหรือจากองค์กร ในช่วงปี 1963 หน่วยงาน ARPA (Advanced Research Projects Agency) ซึ่งเป็นหน่วยงานที่อยู่ภายใต้กระทรวงกลาโหมสหรัฐอเมริกาได้ริเริ่มทำโครงการวิจัยเพื่อพัฒนาระบบเครื่อข่ายคอมพิวเตอร์ที่ใช้สื่อสารระยะไกล โดยให้ชื่อโครงการนี้ว่า ARPANET และในปี 1983 ได้นำโปรโตคอล [[TCP-IP คืออะไร|TCP/IP]] ที่ช่วยให้คอมพิวเตอร์สามารถสื่อสารระหว่างกันได้
-มาใช้งาน จึงได้กลายมาเป็นเครือข่ายอินเทอร์เน็ตที่ใช้งานกันอยู่ในปัจจุบัน
+**Internet** คือระบบเครือข่ายคอมพิวเตอร์ขนาดใหญ่ ที่ประกอบไปด้วยเครือข่ายย่อยทั้งจากส่วนบุคคลหรือจากองค์กร ในช่วงปี 1963 หน่วยงาน ARPA (Advanced Research Projects Agency) ซึ่งเป็นหน่วยงานที่อยู่ภายใต้กระทรวงกลาโหมสหรัฐอเมริกาได้ริเริ่มทำโครงการวิจัยเพื่อพัฒนาระบบเครื่อข่ายคอมพิวเตอร์ที่ใช้สื่อสารระยะไกล โดยให้ชื่อโครงการนี้ว่า ARPANET และในปี 1983 ได้นำโปรโตคอล [[TCP-IP คืออะไร|TCP/IP]] ที่ช่วยให้คอมพิวเตอร์สามารถสื่อสารระหว่างกันได้ มาใช้งาน จึงได้กลายมาเป็นเครือข่ายอินเทอร์เน็ตที่ใช้งานกันอยู่ในปัจจุบัน
```

Comment:
TCP/IP คือ TCP + IP
ถ้า Eng "The Internet protocol suite, commonly known as TCP/IP, is the **set of communications protocols** used in the Internet and similar computer networks."
ไม่ใช่ protocols แต่เป็นรวม protocol

แต่โอเค TCP/IP เยอะสุด


IP ย่อจาก Internet Protocol, IP เดียวกับ IP Address
อะไรที่อยู่บนlayer IP ขึ้นไปถือว่าอยู่บน internet หมด
![1_2B15_AHbTIF8j9VBwmKCtw](https://user-images.githubusercontent.com/8156002/152946441-55d7d16a-cead-4292-9fc3-7d65fcfd96e0.jpg)